templates/stamp/node-vmss.json

```diff
@@ -250,6 +258,44 @@
                                 }
                             }
                         ]
+                    },
+
+                    "extensionProfile": {
+                        "extensions": [
+                            {
+                                "name": "[concat(parameters('networkSettings').clusterVirtualMachineScaleSetName, '-installmdm')]",
+                                "properties": {
+                                    "publisher": "Microsoft.Azure.Extensions",
+                                    "type": "CustomScript",
+                                    "typeHandlerVersion": "2.0",
```

Comment:
Does this mean CustomScript,2.0 worked on Ubuntu14 for the VMSS? Will it work on 14 for the backend as well?